cmd/downloader/downloader/downloadercfg/logger.go

```diff
@@ -13,10 +14,24 @@ func init() {
 	utp.Logger.Handlers = []lg.Handler{noopHandler{}}
 }
 
-func Str2LogLevel(in string) (lg.Level, error) {
+func Int2LogLevel(level int) (lg.Level, error) {
 	lvl := lg.Level{}
-	if err := lvl.UnmarshalText([]byte(in)); err != nil {
-		return lvl, err
+
+	switch level {
+	case 0:
+		lvl = lg.NotSet
+	case 1:
+		lvl = lg.Critical
+	case 2:
+		lvl = lg.Error
+	case 3:
+		lvl = lg.Warning
+	case 4:
+		lvl = lg.Info
```

Comment:
i will merge and change lg.Info to 3 (just to match our --verbosity flag digits).